classes/Gems/MenuNew/Menu.php

```diff
@@ -75,10 +75,14 @@ public function renderContent(): string
 
     public function renderMenu(): string
     {
+        $menu = '';
```

Comment:
Wat is het verschil tussen deze nieuwe code en het opbouwen van de html in `menu-node.html.twig`?